docs/Product-Guides/Benefits-API.md

```diff
@@ -1,40 +1,253 @@
+
+Finch has two types of ways to call our API: Automated and Assisted.
+
+- If the payroll provider is ***automated***, any deductions that are submitted via our API will be changed in the provider’s system instantaneously.
+- If the payroll provider is ***assisted***, any deductions that are submitted via our API will be changed within 7-14 days. This is because a real Finch employee will read your API request and contact the provider directly, manually assist in creating a benefit and enrolling employees in that benefit, and send a response back to you when the request is complete. This benefits you because your team does not need to build a product operations team to make these changes yourself; Finch does this for you.
+
+You can review [our list of valid providers](https://developer.tryfinch.com/docs/reference/96f5be9e0ec1a-providers) to figure out which providers are automated vs assisted for the Benefits endpoints.
+
+## Benefits submissions
+
+Because of the manual nature of assisted benefit connections, it is important to submit payroll benefit deductions to our API 7 days before the customer’s payroll close date. This gives the Finch team a higher chance of making the change in the provider’s system so it can be processed with the current payroll period (unless explicitly specified otherwise in your API request).
+
+For most of your customers, this is already a part of the HR admin workflow. We are just changing the location of where they make this change. It is important as the developer to make it clear to your customers what the payroll close date is so that it can be communicated properly if the benefit will take effect this pay period or next pay period.
+
+## Example
+
+As an example, if the payroll period is **June 1 - 15**. The payroll close date might be **June 16** so payroll can be processed before **Friday, June 17**.  Therefore, it would be important to submit payroll deductions via the Finch API by **June 9** for those to take effect during the June 1 - 15 payroll.
+
+
+<pre>
+June 2022
+Su Mo Tu We Th Fr Sa
+          <mark style="background-color: lightblue">1  2  3  4
+ 5  6  7  8  9 10 11
+12 13 14 15</mark> <mark style="background-color: yellow">16</mark> <mark style="background-color: green">17</mark> 18
+19 20 21 22 23 24 25
+26 27 28 29 30
+</pre>
+
+### Couple of things to keep in mind
+
+- Often there is a company contribution and an employee contribution. Some payroll systems separate these, some add them together when calculating the deduction. When validating if the deduction is successful, make sure to understand how the provider you are working with handles contributions. Our team will add the deduction based on how the provider is set up.
```

Comment:
In our API, we use the terms `company contribution` and `employee deduction`. These terms can get a little confusing. For instance, I'm not sure how the contributions referred to here to affect the deduction in the next sentence. Why would a contribution affect a deduction? It might make sense to clarify the language here.